plugins/vpc-eni/network/network_linux.go

```diff
@@ -13,25 +13,142 @@
 
 package network
 
+import (
+	"github.com/aws/amazon-vpc-cni-plugins/network/imds"
+	"github.com/aws/amazon-vpc-cni-plugins/network/netns"
+
+	log "github.com/cihub/seelog"
+	"github.com/vishvananda/netlink"
+)
+
 // NetBuilder implements the Builder interface for Linux.
 type NetBuilder struct{}
 
 // FindOrCreateNetwork creates a new network.
 func (nb *NetBuilder) FindOrCreateNetwork(nw *Network) error {
+	// A vpc-eni network does not need any network-level setup on Linux.
 	return nil
 }
 
 // DeleteNetwork deletes an existing network.
 func (nb *NetBuilder) DeleteNetwork(nw *Network) error {
+	// A vpc-eni network does not need any network-level cleanup on Linux.
 	return nil
 }
 
 // FindOrCreateEndpoint creates a new endpoint in the network.
 func (nb *NetBuilder) FindOrCreateEndpoint(nw *Network, ep *Endpoint) error {
```

Comment:
Do we also need to set MTU for the interface? I think ecs-eni plugin supports customizing the MTU.

https://github.com/aws/amazon-ecs-cni-plugins/blob/master/plugins/eni/engine/nsclosure.go#L142

Reply:
Yes, ecs-eni supports custom MTUs. However I see no requirement to do so in vpc-eni. ENIs are VPC constructs and should use VPC MTUs.